packages/react-icons/src/utils/createFluentIcon.ts

```diff
@@ -9,9 +9,10 @@ export type FluentIcon = {
 
 export const createFluentIcon = (displayName: string, width: string, paths: string[]): FluentIcon => {
     const viewBoxWidth = width === "1em" ? "20" : width;
-    const Icon = (props: FluentIconsProps) => {
+    const Icon = React.forwardRef((props: FluentIconsProps, ref: React.Ref<SVGSVGElement>) => {
```

Comment:
changes we discussed offline

```suggestion
    const Icon = React.forwardRef((props: FluentIconsProps, ref: React.Ref<SVGElement>) => {
```